usaspending_api/search/models/base_award_search.py

```diff
@@ -19,6 +19,8 @@ class BaseAwardSearchModel(models.Model):
     category = models.TextField()
```

Comment:
missing *_agg_key fields in the base model

Reply:
These have been added